### ISSUE TITLE
"Local Keycloak setup" instructions not fully clear

### ISSUE THREAD

**Issue by P4ul1029** [CDH-Studio/UpSkill#1651]
**Describe the issue**
I was instructed to test the _Local Keycloak setup_ instructions, but it is not clear what changes step 3 from the instructions are referencing. In addition, after going through the instructions, trying to access http://localhost:8180/auth results in a ERR_CONNECTION_REFUSED error.

**Expected behavior**
After completing each step from the instructions, Keycloak should be able to authenticate users locally.

**Comment by P4ul1029** [comment]
Step 1 from the instructions says to add `network_mode: "host"` to the backend container in _docker-compose.yml_. This can not be done because it is [incompatible with port mapping](https://docs.docker.com/compose/compose-file/compose-file-v3/#ports).

Step 3 references the usage of [links](https://docs.docker.com/compose/compose-file/compose-file-v3/#links) which are not present in the _docker-compose.yml_ file.

For step 2, I added the following environment parameters to the outlined keycloak service in order to resolve errors where the service could not connect to the postgres database:

```
DB_VENDOR: postgres
DB_ADDR: postgres
DB_PORT: 5432
DB_DATABASE: testdb
DB_USER: api
DB_PASSWORD: api
```

After applying those changes and updating the docker containers, I was able to access http://localhost:8180/auth and log into I-Talent with the users outlined in the _realm-export.json_ file. However, after logging in, the request to `:8080/api/user` returns a 403 forbidden error.
